vnext/template/shared-app/proj/ExperimentalFeatures.props

```diff
@@ -17,6 +23,12 @@
       See https://microsoft.github.io/react-native-windows/docs/winui3
     -->
     <UseWinUI3>{{useWinUI3}}</UseWinUI3>
+
+    <!--
+      Changes compilation to assume use of Micorsoft.ReactNative NuGets packages instead of source.
```

Comment:
typo ;)
```suggestion
      Changes compilation to assume use of Microsoft.ReactNative NuGet packages instead of source.
```

<!--StartFragment-->      Changes compilation to assume use of Micorsoft.ReactNative NuGets packages instead of source.<!--EndFragment-->



<!--StartFragment-->      Changes compilation to assume use of Micorsoft.ReactNative NuGets packages instead of source.<!--EndFragment-->